tuf/repository_lib.py

```diff
@@ -750,13 +750,13 @@ def _log_warning_if_expires_soon(rolename, expires_iso8601_timestamp,
   """
  
   # Metadata stores expiration datetimes in ISO8601 format.  Convert to
-  # unix timestamp, subtract from from current time.time() (also in POSIX time)
+  # unix timestamp, subtract from from current time (also in POSIX time)
```

Comment:
Not really part of your change, but while you're at the line... s/from from/from/